internal/page/supporter/delete_organisation.go

```diff
@@ -0,0 +1,45 @@
+package supporter
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/ministryofjustice/opg-go-common/template"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/actor"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/page"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/validation"
+)
+
+type deleteOrganisationData struct {
+	App                page.AppData
+	Errors             validation.List
+	InProgressLPACount int
+}
+
+func DeleteOrganisation(tmpl template.Template, organisationStore OrganisationStore, sessionStore SessionStore) Handler {
+	return func(appData page.AppData, w http.ResponseWriter, r *http.Request, organisation *actor.Organisation) error {
+		lpas, err := organisationStore.AllLPAs(r.Context())
```

Comment:
I forgot to delete this method when doing the paging, this should call OpenSearch for the result as this query on dynamo is bad